08_deploy_bmo.sh

```diff
@@ -4,13 +4,14 @@ eval "$(go env)"
 
 # Get the latest bits for baremetal-operator
 export BMOPATH="$GOPATH/src/github.com/metalkube/baremetal-operator"
+export KUBECONFIG="ocp/auth/kubeconfig"
 
-oc --as system:admin --config ocp/auth/kubeconfig new-project bmo-project
+oc new-project bmo-project
 
 # Start deploying on the new cluster
 
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/service_account.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/role_binding.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/crds/metalkube_v1alpha1_baremetalhost_crd.yaml
-oc --as system:admin --config ocp/auth/kubeconfig apply -f $BMOPATH/deploy/operator.yaml
+oc apply -f $BMOPATH/deploy/service_account.yaml
```

Comment:
@sadasu Hey was there a reason you needed --as system:admin when using the admin kubeconfig?

Reply:
Let me try without it. I had introduced it early on when I had encountered several issues one of them having something to do with authentication failing.

Reply:
@sadasu did you have the chance to try this?

Reply:
@e-minguez apologies for not updating this thread. I had tried it and updated my patch. Thanks for pointing that out.